source/_patterns/20-molecules/comments/comments.scss

```diff
@@ -549,21 +552,23 @@
 
   &:focus {
     outline: none;
+
     + .reply-info .reply-info__count {
       position: static;
       opacity: 1;
     }
   }
 }
 
-.reply__label, reply__help {
+.reply__label,reply__help {
```

Comment:
Is there a `.` missing? 🤔

Reply:
yes, but that was missing before. I didn't want to fix it in this issue, because this would need more testing effort. but at least, it will be visible in the linter output until someone fixes it... ;-)

Reply:
https://github.com/mmz-srf/srf-frontend-framework/commit/96755bf88cd2137cf26922f9a594583f2ee8da42

hust hust